app/controllers/tag_controller.rb

```diff
@@ -514,10 +514,12 @@ def stats
   end
 
   def comments
+    fetch_counts
     tids = Tag.where(name: params[:id]).collect(&:tid)
     nids = NodeTag.where('tid IN (?)', tids).collect(&:nid)
     @pagy, @comments = pagy(Comment.where(nid: nids).order('timestamp DESC'), items: 24)
-    render template: 'comments/_comments', locals: { comments: @comments }
+    @node_type = "comments"
+    render 'show', locals: { comments: @comments }
```

Comment:
@billymoroney1 Since we already have an instance variable named `@comments`, we don't need to pass it as a local variable
```suggestion
    render 'show'
```